### PR TITLE
KAFKA-14588 [3/N] ConfigCommandTest rewritten in java

### PR DESCRIPTION
**Pull request by nizhikov**
This is third part of https://github.com/apache/kafka/pull/15839 refactoring.
PR intention - split big PR in parts to simplify review.
PR contains tests of ConfigCommandTest rewritten in java.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by nizhikov** [comment]
Hello @chia7712 

I opened PR with third part of `ConfigCommandTest` refactoring.
Can you, please, take a look?

**Comment by nizhikov** [comment]
Added tests passed. PR ready for review.

File: core/src/test/java/kafka/admin/ConfigCommandUnitTest.java [hunk 1]
```diff
@@ -878,6 +886,486 @@ public void shouldNotDescribeUserScramCredentialsWithEntityDefaultUsingBootstrap
...
+            @Override
+            public void changeTopicConfig(String topic, Properties configChange) {
+                assertEquals("my-topic", topic);
+                assertEquals("b", configChange.get("a"));
+                assertEquals("d", configChange.get("c"));
+            }
+        });
+    }
+
+    @Test
```
Comment (chia7712) [inline core/src/test/java/kafka/admin/ConfigCommandUnitTest.java]: How about using `ValueSource`?

```java
    @Test
    @ValueSource(booleans = {true, false})
    public void shouldAlterTopicConfig(boolean file) {
```

**Review by chia7712 — COMMENTED** [review]
@nizhikov nice patch. one small comment is left.

File: core/src/test/java/kafka/admin/ConfigCommandUnitTest.java (hunk 1 above)
Comment (nizhikov) [reply core/src/test/java/kafka/admin/ConfigCommandUnitTest.java]: Done.

**Comment by nizhikov** [comment]
CI looks Ok for me

**Comment by chia7712** [comment]
@nizhikov nice patch!

**Comment by nizhikov** [comment]
@chia7712 Thanks for the review and merge.

**Pull request merged**